cluster/gke/config-test.sh

```diff
@@ -18,6 +18,7 @@
 CLUSTER_NAME="${CLUSTER_NAME:-${USER}-gke-e2e}"
 NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
+IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-}"
```

Comment:
I've been following the convention that other env vars are defined: test overrides are always prefixed with `KUBE`. For example: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh

Also, I don't have the permission to add labels.